scripts/sundialsdev/logs.py

```diff
@@ -72,13 +82,12 @@ def parse_logfile_line(line, line_number, all_lines):
     return line_dict
 
 
-def log_file_to_list(filename, step_scope_txt):
+def log_file_to_list(filename):
     """
-    This function takes a debug log file from a SUNDIALS log file and creates a list where
-    each list element represents an integrator step attempt.
+    This function takes a SUNDIALS log file and creates a list where each list
+    element represents an integrator step attempt.
 
     E.g.,
-    [
```

Comment:
This function no longer captures all the log entries between "begin-step-attempt" and "end-step-attempt". If that was your intent, then it shuold be a different function.

Reply:
This function was specifically to capture *all* log output and organize into a list of lists, where each inner list holds all the logging output lines (each one as a dictionary) for a step attempt.

Reply:
For a nested integrator, I think it should be like:
```
[
  // first level is outer steps
   [
      // outer step 1
     { stuff },
     { stuff },
     [ 
        // inner step
     ],
   ],
]
```

Reply:
The intent is to handle all the entries but I haven't gotten there yet (working on making it easier to parse/plot the nonlinear/linear solver logging first). 

I originally tried the structure above for MRI output but it was difficult to associate the fast integrations with their corresponding stages at the slow time scale. The structure I ended up going with is a nesting lists of step attempt data with in lists of stage data.

For example an RK method the list is
```
[
  {
    'step' : 1,
    't_n' : 0,
    ...
    'stages' :
    [
      { 
         'stage' : 0
         ...
      },
      {
         'stage' : 1,
         ...
      }
    ],
   'dsm' : 2.363391824501069e-08
  },
 ...
]     
```
For an MRI method each dictionary in the `stages` list has a `time-level-#` entry that is a list of step attempt dictionaries just like above.